tests/neo4j/suites.py

```diff
@@ -90,6 +102,9 @@
         print("Unknown suite name: " + name)
         sys.exit(-1)
 
+    import os
+    os.environ["env_neo4j_version"] = os.environ.get("env_neo4j_version", name)
```

Comment:
Out of interest, how is this used?

Reply:
This, together with this https://github.com/neo4j-drivers/testkit/pull/156/files#diff-81d6da2c1420e0c0ab1903e78fec06d8ab0d78483f4e47abe41c82cd2aa4ef36R65-R92 is used to know which features can be used in tests (e.g. multi-database).